docs/explorer.rst

```diff
@@ -305,6 +307,7 @@ Once we have a `Surface` object we can get surface metadata using properties:
     print(surfaces.uuid)
     print(surfaces.name)
     print(surfaces.tagname)
+    print(surfaces.dataformat)
```

Comment:
Should this be `dataformats`?

Reply:
No, but all the "surfaces" should be "surface". Fixed ✅